### ISSUE TITLE
Release for Keycloak 26

### ISSUE THREAD

**Issue by daviddavidgit** [adorsys/keycloak-config-cli#1220]
### Problem Statement

There is no release for Keycloak 26. Is there a reason why?

**Comment by jwklijnsma** [comment]
@daviddavidgit you can used edge image is working for mee

**Comment by gtudan** [comment]
Hi! We really love the work on this project - Keycloak would be way less fun without the config-cli!
Some transparency on the release plan would be highly appreciated - is there a certain feature that you are working on that should go into the release? If so, would it be possible to have a minor release in between?
Keycloak 26 has been out for a long time and we are already overdue for an update. But we had to postpone it over and over because of the blocking issues already fixed in #1160.

**Comment by rblaine95** [comment]
Looks like `6.2.0` has been tagged and it includes Keycloak `26.0.5`

* https://github.com/adorsys/keycloak-config-cli/releases/tag/v6.2.0

* https://hub.docker.com/layers/adorsys/keycloak-config-cli/6.2.0-26.0.5/images/sha256-342fbd3d32a60c63fca6568667ba73dfeb12c7e54831493f6dd7940e6b713327?context=explore

**Comment by gtudan** [comment]
Yes! Looks like @st3v0rr  is on it. Unfortunately the CI failed, so no docker images available yet.
https://github.com/adorsys/keycloak-config-cli/actions/runs/12135604720

**Comment by st3v0rr** [comment]
Hello,
we have currently some trouble with the github actions for artifact upload and download.
I will ask the team to investigate some time here.
Same happened in July but the rollback to the current versions helped. Now it looks like this is also broken :-(

**Comment by dguodev** [comment]
Hi team:

sorry to follow up , any estimate when 26 will be release to docker repo ?

**Comment by tanadeau** [comment]
With the release of 6.2.1, can this be closed?
